AdyenTests/Card Tests/Utilities/CardTypeProviderTests.swift

```diff
@@ -1,9 +1,7 @@
 //
-//  CardTypeProviderTests.swift
-//  AdyenTests
+// Copyright (c) 2020 Adyen N.V.
 //
-//  Created by Vladimir Abramichev on 24/08/2020.
-//  Copyright © 2020 Adyen. All rights reserved.
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
 //
 
 @testable import AdyenCard
```

Comment:
empty file :-)

Reply:
what do you mean?

Reply:
sorry, I mean, why change the file header comments?

Reply:
it is nicer :D